src/client/common/types.ts

```diff
@@ -227,6 +227,12 @@ export interface IMypyCategorySeverity {
     readonly error: DiagnosticSeverity;
     readonly note: DiagnosticSeverity;
 }
+
+export type LoggingLevelSettingType = 'off' | 'error' | 'warn' | 'info' | 'debug';
+
+export interface ILoggingSettings {
+    readonly level: LoggingLevelSettingType;
+}
```

Comment:
`LoggingLevelSettingType` should be the encoding of the data in settings.json.  However, in the extension we should be using `logging.levels.LogLevel` everywhere.  So:

```suggestion
export type LoggingLevelSettingType = 'off' | 'error' | 'warn' | 'info' | 'debug';

export interface ILoggingSettings {
    readonly level: LogLevel;
}
```

Reply:
`ILoggingSettings` refers to defined the type of data in settings.json alone. It's no longer used in the extension.